lib/jq.js

```diff
@@ -1,4 +1,24 @@
 const nativeJq = require('bindings')('jq-node-bindings')
+nativeJq.setCacheSize(2000)
+
+let pLimit = null
+let currentLimit = 2
+
+const initPLimit = async () => {
+  if (!pLimit) {
+    const module = await import('p-limit')
+    pLimit = module.default(currentLimit)
+  }
+  return pLimit
```

Comment:
Should it be like this so we won't initialize the pLimit object on every request?

```suggestion
  return pLimit(currentLimit)
```